coreos-base/oem-packet/files/reset-interfaces.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
yeah, I didn't mean to imply rackspace should be copied, the issue is slightly different. Just add the set -e